cnd/src/swap_protocols/facade.rs

```diff
@@ -26,7 +26,7 @@ pub struct Herc20HalightBitcoinCreateSwapParams {
     pub ethereum_absolute_expiry: Timestamp,
     #[digest(prefix = "2002")]
     pub ethereum_amount: asset::Erc20Quantity,
-    #[digest(ignore)]
+    #[digest(prefix = "2003")]
     pub token_contract: identity::Ethereum,
```

Comment:
Would be nice if we can change `identity::Ethereum` to some other type (alias) sometimes, as semantically this is not an identity, but a token contract address. 
Can do that in another iteration.